modules/4337/certora/requirements.txt

```diff
@@ -1 +1,2 @@
-certora-cli==7.14.3
+# TODO: remove beta when 7.21.0 is stable
```

Comment:
TODO: Merge when 7.21.0 is stable and switch back to certora-cli instead of the beta version